.travis.yml

```diff
@@ -3,4 +3,4 @@ sudo: false
 language: node_js
 
 node_js:
-  - "6.10.2"
+  - "10.5.0"
```

Comment:
You can, and should, remove the `node_js` config completely. Travis uses the `.nvmrc`.